sound/soc/sof/intel/Kconfig

```diff
@@ -26,6 +26,8 @@ config SND_SOC_SOF_INTEL_ATOM_HIFI_EP
 config SND_SOC_SOF_INTEL_COMMON
 	tristate
 	select SND_SOC_SOF
+	select SND_SOC_SOF_IPC3
+	select SND_SOC_SOF_INTEL_IPC4
```

Comment:
this could be done in a better way, for legacy ACPI devices we only support IPC3. For newer platforms, we support IPC4 as a backup. I think this needs to be moved down a level.

Reply:
@plbossart, OK, I'll move the IPC version selection per platform to narrow it down (my Gentoo user self will thank this ;) )